src/services/TileService.ts

```diff
@@ -389,7 +391,7 @@ export class TileService {
 
     private clearTile = (tile: RasterTile, _key: any) => {
         if (tile.data) {
-            delete tile.data;
+            tile.data = new Float32Array();
```

Comment:
If we assign the tile data to empty array, will it allocate memory space to it?

Reply:
Good point... I revised to fix it in a safer way.